### DISCUSSION TITLE
Why the type for `useEnvironmentContext` is not named `UseEnvironmentContext`, instead named `EnvironmentContext`?

### DISCUSSION THREAD

**Discussion by babakfp** [chakra-ui/ark#3082] [Q&A] [answered]
Hi

`packages\solid\src\providers\environment\use-environment-context.ts`

As far as I've seen, types like this are prefixed with `Use` in the source code, but for this one it's differnt. Is there a reason, or is this a typo?

Thanks

**Comment by cschroeter** [comment, marked as answer]
@babakfp - thanks for reaching out. I've deprecated `EnvironmentContext` in favor of `UseEnvironmentContext`. This new type will be available in the next release and the previous one will be removed in the next major. Cheers
